client/sttp-client/src/main/scala/sttp/tapir/client/sttp/package.scala

```diff
@@ -1,3 +1,9 @@
 package sttp.tapir.client
 
-package object sttp extends TapirSttpClient
+package object sttp {
+  def apply(clientOptions: SttpClientOptions = SttpClientOptions.default): TapirSttpClient = {
```

Comment:
hm I think this might be confusing. Maybe it's time to remove the deprecated `TapirSttpClient` simply